packages/expo-optimize/package.json

```diff
@@ -36,6 +36,7 @@
     "@expo/config": "6.0.7",
     "@expo/image-utils": "0.3.17",
     "@expo/json-file": "8.2.33",
+    "@supercharge/promise-pool": "^1.8.0",
```

Comment:
this dependency doesn't look particularly reliable and it's much larger than it needs to be. can you replace this with p-limit instead? https://www.npmjs.com/package/p-limit